platform/ios/PolyPodApp/PodApi/Network/Network.swift

```diff
@@ -34,29 +45,88 @@ class Network: NetworkProtocol {
         
         let semaphore = DispatchSemaphore(value: 0)
         var errorMessage: String? = nil
+        var responseData: String? = nil
+        var responseCode: Int = 400
         let task = URLSession.shared.dataTask(with: request) {
             data, response, error in
             guard let response = response as? HTTPURLResponse,
                   error == nil else {
-                errorMessage = error?.localizedDescription ?? "Unknown error"
+                      errorMessage = error?.localizedDescription ?? "Unknown error"
```

Comment:
Nit: Indentation seems off now, should be at the same level as the guard block below, no?

Reply:
Yeah it's what xCode wants so I guess you have no power here